picturae_csv_create.py

```diff
@@ -256,11 +256,14 @@ def taxon_concat(self, row):
                 if "var." in full_name or "subsp." in full_name or " f." in full_name or "subf." in full_name:
                     hybrid_base = full_name
                     full_name = " ".join(taxon_strings[:2])
-                else:
+                elif full_name != genus and full_name == gen_spec:
```

Comment:
made the elif condition more specific and able to better handel genus level vs species level hybrids